src/main/resources/db/migration/V23__slett_statistikk_eldre_enn_2017.sql

```diff
@@ -0,0 +1,2 @@
+delete from sykefravar_statistikk_virksomhet_med_gradering where arstall < 2017;
+delete from sykefravar_statistikk_virksomhet where arstall < 2017;
```

Comment:
Skal vi bare fjerne for virksomheter? Eller for næring, sektor og land også ?

Reply:
Tok en prat med M nå, hun tenker vi like godt kan slette alt sammen, så da skal jeg oppdatere